### ISSUE TITLE
Accessibility: Empty headings

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#524]
**Describe the bug**
Empty headings cannot exist for accessibility. All heading tags must have content.

Page: [SiteImprove Exhibits: Empty Headings](https://my2.siteimprove.com/Accessibility/1081272/NextGen/Issue?ruleId=64&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteGoal=)

**To Fix**
Content Owners will need to be sure that headings exist for all images in slideshows and on all pages. Headings are currently missing on 5 different pages. This should be fixed by Content Owners.

**Pages to fix:**
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68309665128&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r64/failed/WWRtNld6a2o=;Mjc5YmVkNjY=
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58322089057&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r64/failed/dVl1Q2ZFUlA=;ZWFmODgzNTU=
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68309665135&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r64/failed/WWRtNld6a2o=;Mjc5YmVkNjY=
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58325076337&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r64/failed/R1I3djBWeWE=;ZTkyNzNkYmE=
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68309665131&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r64/failed/WWRtNld6a2o=;Mjc5YmVkNjY=
